packages/account/src/BiconomySmartAccount.ts

```diff
@@ -32,53 +33,64 @@ export class BiconomySmartAccount extends SmartAccount implements IBiconomySmart
 
   constructor(readonly biconomySmartAccountConfig: BiconomySmartAccountConfig) {
     const {
-      signerOrProvider,
+      signer,
       rpcUrl,
       epAddress,
       factoryAddress,
       bundlerUrl,
       paymasterUrl,
-      nodeClientUrl
+      nodeClientUrl,
+      dappApiKey,
```

Comment:
have resolved this. Instead of paymasterUrl and bundlerUrl. I am accepting instances for both paymaster and bundler. here is the update type of BiconomySmartAccountConfig

```
export type BiconomySmartAccountConfig = {
  signer: Signer,
  rpcUrl?: string,
  chainId: ChainId
  entryPointAddress?: string
  factoryAddress?: string
  bundler?: IBundler
  paymaster?: IPaymaster
  nodeClientUrl?: string
}
```